ui/app/store/store.js

```diff
@@ -1,21 +1,18 @@
-const createStore = require('redux').createStore
-const applyMiddleware = require('redux').applyMiddleware
-const thunkMiddleware = require('redux-thunk').default
+const { createStore, applyMiddleware } = require('redux')
+const { default: thunkMiddleware } = require('redux-thunk')
+const { composeWithDevTools } = require('remote-redux-devtools')
 const rootReducer = require('../ducks')
-const createLogger = require('redux-logger').createLogger
 
-global.METAMASK_DEBUG = process.env.METAMASK_DEBUG
-
-module.exports = configureStore
-
-const loggerMiddleware = createLogger({
```

Comment:
I assume you deleted this middleware intentionally? I suppose it is superseded by the plugin. Seems reasonable to remove it.

Reply:
Yes, the logger middleware was console noise and the Redux DevTools is the place for that now